components/remerge/src/storage/bootstrap.rs

```diff
@@ -36,8 +36,27 @@ pub(super) fn load_or_bootstrap(
 
         if native_ver != native.version.to_string() {
             // XXX migrate existing records here!
-            // XXX Ensure we only move native version forward and not backwards!
+            let native_ver = semver::Version::parse(&*native_ver)
+                .expect("previously-written version is no longer semver");
```

Comment:
This did not seem worth surfacing as an error constructor, but `.expect()` here isn't great. Thoughts?